gc/base/omrmmprivate.hdf

```diff
@@ -984,6 +984,32 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<data type="void *" name="concurrentStats" description="internal statistics for concurrent phase" />
```

Comment:
`J9HOOK_MM_PRIVATE_CONCURRENT_PHASE_END` still used in `VerboseHandlerOutputVLHGC`.  I guess We could pass either of the new triggers and the end result will be the same. Just it will be bit confusing.

Reply:
I think we initially discussed having these new triggers used for VLGC as well. There will be 2 concurrent cycles down the road there.

Reply:
The existing one can be changed to the new `J9HOOK_MM_PRIVATE_CONCURRENT_MARK_PHASE_END`

Reply:
`J9HOOK_MM_PRIVATE_CONCURRENT_MARK_PHASE_END` -> `J9HOOK_MM_PRIVATE_CONCURRENT_GLOBAL_PHASE_END `
`J9HOOK_MM_PRIVATE_CONCURRENT_SCAVENGE_PHASE_END` -> `J9HOOK_MM_PRIVATE_CONCURRENT_LOCAL_PHASE_END`